source/objects/lander.lua

```diff
@@ -500,10 +501,12 @@ function Lander.draw(worldOffset)
 			love.graphics.setColor(1,1,1,0.5)
 		end
 
-		local ox = lander.width / 2
-		local oy = lander.height / 2
+		-- TODO: for some reason ox = lander.width fails because lander.width is sometimes nil.
```

Comment:
That's something that never should happen. Since I've never experienced it to fail it might has to do with socketstuff or possibly save/load game since I've never used any of this when playtesting.

Reply:
Yes. Is a socket thing.